apps/meteor/ee/app/livechat-enterprise/server/services/omnichannel.internalService.ts

```diff
@@ -40,9 +48,7 @@ export class OmnichannelEE extends ServiceClassInternal implements IOmnichannelE
 		if (room.onHold) {
 			throw new Error('error-room-is-already-on-hold');
 		}
-		if (room.lastMessage?.token) {
-			throw new Error('error-contact-sent-last-message-so-cannot-place-on-hold');
-		}
+
```

Comment:
The other ifs don't have an space in between.

Since the change was to remove a condition, you can remove this white line